src/web/admin.rs

```diff
@@ -82,11 +82,11 @@ async fn handler(
                 .read_to_string(&mut read_body)?;
 
             match target_runtime
-                .spawn(add_configuration(
-                    state,
-                    "POST web configuration".to_string(),
-                    read_body,
-                ))
+                .spawn(async {
```

Comment:
spawn_blocking?

Reply:
`spawn_blocking` doesn't exists on `Runtime` 😞 
It's for this reason I `spawn` then `block_in_place`